understanding/21/non-text-contrast.html

```diff
@@ -268,18 +262,36 @@ <h4>User Interface Component Examples</h4>
 							<td><img src="img/checkbox-example2.png" alt="Black square border with a tick inside, and a text label." width="150"/></td>
 						</tr>
 						<tr>
-							<th>Checkbox - Fail</th>
-							<td>The grey border color of the checkbox (#9D9D9D) has a contrast ratio of 2.7:1 with the white background, which is not sufficient for the visual information required to identify the checkbox.</td>
-							<td><img src="img/checkbox-example3.png" alt="Grey box on a white background with a black tick in the middle." width="150"/></td>
+							<th>Checkbox - Subtle hover style</th>
+							<td>A black border on a white background indicates the checkbox, when the mouse pointer activates the subtle hover state adds a grey background (<code>#DEDEDE</code>). The black border has a 15:1 contrast ratio with the grey background.</td>
```

Comment:
```suggestion
							<td>A black border on a white background indicates the checkbox, but when the mouse pointer hovers on the checkbox, a subtle grey background is added (<code>#DEDEDE</code>). The black border has a 15:1 contrast ratio with the grey background, so the hover effect clearly does not cause the checkbox to fail contrast. Note that the grey hover effect does not itself need to contrast 3:1 with the page background, since the pointer position is the primary indicator of the hover state.</td>
```

Reply:
This sentence needs some work, I think:

> so the hover effect clearly does not significantly reduce the checkbox indicator's contrast

1. I'd remove the word "clearly" — it's likely not clear to some people
2. "does not significantly reduce the checkbox indicator's contrast":
    a. What does "significantly" mean? To reduce the contrast to less than 3:1?
    b. What's the "indicator"? Is it the border?

What about:

"The black border has a 15:1 contrast ratio with the grey background, which is above the required 3:1 ratio."